fix(curriculum): remove double punctuation after inline code and quotes (#65730)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-emoji-reactor/6899b0d1825ff24e9fe8d747.md

assert.equal(document.querySelector("h1 + p")?.className, "description");
```

> The new `p` element should have text of `Click on the buttons below to rate your emotions.`

Reviewer: ```suggestion
The new `p` element should contain the text: `Click on the buttons below to rate your emotions.`
```
